packages/cms-lib/links.js

```diff
@@ -0,0 +1,138 @@
+const { getEnv } = require('./lib/config');
```

Comment:
I think we should keep this in `cms-cli` since it is unlikely to be useful outside the CLI.  Moving to CLI has the added benefit that it reduces the dependencies needed in `@hubspot/cms-lib`.